src/utils/tools.ts

```diff
@@ -8,11 +8,11 @@ import { Options, resources, runUntyped } from '../askvm';
 const myLogger = util.debuglog('');
 
 export async function e2e(
-  script: string,
+  askScriptCode: string,
   environment: Options,
   args?: any[]
 ): Promise<any> {
-  const ast = parseToAst(script);
+  const ast = parseToAst(askScriptCode);
```

Comment:
I agree that this is a good idea to add more information about this argument. In TypeScript we can create custom types so we can used that as a handy alias:

```ts

type AskScriptCode = string;

export async function e2e(
  script: AskScriptCode,
```

This way we won't need to change any of the implementation - could be done safely in the same PR

Reply:
OK, I am creating `type AskScriptCode`